webview-ui/vite.config.ts

```diff
@@ -35,5 +35,6 @@ export default defineConfig({
 	},
 	define: {
 		"process.platform": JSON.stringify(process.platform),
+		"process.env.VSCODE_TEXTMATE_DEBUG": JSON.stringify(process.env.VSCODE_TEXTMATE_DEBUG),
```

Comment:
Shiki (https://github.com/shikijs/shiki) uses this ENV var for debugging, so we need pass it through or it complains.